assets/js/components/Dashboard/AssociateAlertPage.tsx

```diff
@@ -0,0 +1,327 @@
+import React, { ComponentType, useState } from "react";
+import {
+  Button,
+  Container,
+  Row,
+  Col,
+  ButtonGroup,
+  Modal,
+  Form,
+} from "react-bootstrap";
+import { useNavigate } from "react-router-dom";
+import { useScreenplayContext } from "../../hooks/useScreenplayContext";
+import {
+  useAssociatedAlertContext,
+  useAssociatedAlertDispatchContext,
+} from "../../hooks/useScreenplayContext";
+import { Alert } from "../../models/alert";
+import classNames from "classnames";
+import { getAlertEarliestStartLatestEnd } from "../../util";
+
+const AssociateAlertPage = () => {
+  const navigate = useNavigate();
+  const {
+    associatedAlert,
+    endWithEffectPeriod: endWithEffectPeriodState,
+    importLocations: importLocationsState,
+    importMessage: importMessageState,
+  } = useAssociatedAlertContext();
+  const { alerts } = useScreenplayContext();
```

Comment:
I'm realizing now that we might need to fetch these alerts separately on page load. The alerts in context filter to only alerts that can show on showtime screens. This rules out a large number of effects and also all of Mattapan and Silver Line. It also includes Access alerts which we don't want here.

Reply:
I see. There's a not about deferring any alert filtering to a later task, but that's just for Access alerts. We certainly shouldn't rule out Mattapan and Silver Line in this task